lib/travis/build/script/rust.rb

```diff
@@ -12,7 +12,7 @@ class Rust < Script
         def export
           super
 
-          sh.export 'TRAVIS_RUST_VERSION', config[:rust].to_s.shellescape, echo: false
+          sh.export 'TRAVIS_RUST_VERSION', version.shellescape, echo: false
```

Comment:
`version` method is here.
https://github.com/travis-ci/travis-build/blob/c5dab5e13794e1c597e810dfa5db40c2eb70bcfa/lib/travis/build/script/rust.rb#L63-L65